agrirouter-sdk-java-impl/src/main/java/com/dke/data/agrirouter/impl/messaging/MessageEncoder.java

```diff
@@ -232,15 +220,40 @@ default EncodedMessage encode(SetCapabilitiesParameters parameters) {
     return new EncodedMessage(applicationMessageID, encodedMessage);
   }
 
+  static void setSequenceNumber(
```

Comment:
Wouldn't this better be a member of `MessageHeaderParameters`?

Reply:
I do not think so, this would mean, that we are providing the business objects within the parameter class (means providing the onboard response to the parameters). I would like to keep the code clean and the objects closer together to reduce dependencies between objects.